subsys/bluetooth/controller/ll_sw/ti/cmd-ble-slave.txt

```diff
@@ -0,0 +1,173 @@
+*** Booting Zephyr OS build zephyr-v2.2.0-2078-g1924f672b464  ***
```

Comment:
Looks like a stray file that has been committed by mistake.
If this is added on purpose, please give reason.

Reply:
This is still very much WIP. It's here for the remote TI team on the other side of the planet to have a reference :-) I will definitely remove once the PR is in ready state.